lib/queue.js

```diff
@@ -884,6 +884,11 @@ Queue.prototype.retryJob = function(job) {
 };
 
 Queue.prototype.toKey = function(queueType){
+  return [this.keyPrefix ? '' : 'bull', this.name, queueType].join(':');
+};
+
+Queue.prototype.toPrefixedKey = function(queueType){
+  if (!this.keyPrefix) return this.toKey(queueType);
   return [this.keyPrefix, this.name, queueType].join(':');
 };
 
```

Comment:
It might not be a good idea because:

- the name 'redisOptions' would become misleading
- it would confuse the redisOptions.createClient API